CONTRIBUTING.md

```diff
@@ -0,0 +1,282 @@
+# How to contribute
+
+## How to prepare
```

Comment:
to prepare what? I guess a contribution. Maybe also say why is that necessary. I guess to prevent duplicate work (and in that case, you'll also need the issue to be assigned to a person)